templates/support/Configuration.swift.hbs

```diff
@@ -5,29 +5,16 @@
 import Foundation
 
 public typealias ConfigurationAPIKeyBlock = ((_ name: String) -> String?)
-public typealias ConfigurationAuthorizationBlock = ((_ name: String, _ scopes: [String]?) -> String?)
+public typealias ConfigurationAuthorizationBlock = ((_ name: String, _ scopes: [String]?, _ request: URLRequest) -> URLRequest)
```

Comment:
Instead of `String` can't we use the `SecurityScheme` type?

Reply:
we have deleted it